<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>17</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>LISTAR CLIENTES</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabela_dados">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>531</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 79);
color: rgb(0, 0, 0);</string>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOME</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EMAIL</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ENDEREÃ‡O</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="bt_deletar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>410</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(170, 0, 255);</string>
    </property>
    <property name="text">
     <string>DELETAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_editar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(170, 0, 255);</string>
    </property>
    <property name="text">
     <string>EDITAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
